news/2025/2025-09-17-osumania-4k-world-cup-2025-quarterfinals-recap.md

```diff
@@ -0,0 +1,153 @@
+| 6 | Spain ::{ flag=ES }:: | ::{ flag=PE }:: Peru | LN2 | 907 | ::{ flag=PE }:: | ::{ flag=PE }:: | [![#1](/wiki/shared/link/Link.png)](https://osu.ppy.sh/community/matches/119207311 "Match history") |
+| 7 | Malaysia ::{ flag=MY }:: | ::{ flag=ID }:: Indonesia | RC2 | 1,016 | ::{ flag=ID }:: | ::{ flag=ID }:: | [![#1](/wiki/shared/link/Link.png)](https://osu.ppy.sh/community/matches/119193826 "Match history") |
+| 8 | Thailand ::{ flag=TH }:: | ::{ flag=JP }:: Japan | RC7 | 1,194 | ::{ flag=JP }:: | ::{ flag=TH }:: | [![#1](/wiki/shared/link/Link.png)](https://osu.ppy.sh/community/matches/119202981 "Match history") |
+| 9 | Argentina ::{ flag=AR }:: | ::{ flag=ES }:: Spain | RC3 | 1,439 | ::{ flag=AR }:: | ::{ flag=ES }:: | [![#1](/wiki/shared/link/Link.png)](https://osu.ppy.sh/community/matches/119196396 "Match history") |
+| 10 | Hong Kong ::{ flag=HK }:: | ::{ flag=GB }:: United Kingdom | RC4 | 1,605 | ::{ flag=HK }:: | ::{ flag=HK }:: | [![#1](/wiki/shared/link/Link.png)](https://osu.ppy.sh/community/matches/119206077 "Match history") |
+
+At first glance you may notice that the match between ::{ flag=TH }:: Thailand and ::{ flag=JP }:: Japan had not one, not two, but three of the top 10 closest maps throughout the whole round. While going off of the final result it may not look like it, that final scoreline could have been very different.
+
+Thailand picking their protect of [Ariabl'eyeS - Romanesque Romancia](https://osu.ppy.sh/beatmapsets/2428522) (RC3) as the first map of the match ended up being a point for Japan. An unfortunate miss about halfway through the map gave Japan the lead, and from there, they just about managed to hold on even though the accuracies favoured the Thai roster. Right after that map, the Japanese protect of [Puru - Toki (maidable Remix)](https://osu.ppy.sh/beatmapsets/1306576) (RC7) ended up going to Thailand with a single miss on both sides. This resulted in a close point right up until the very end.
+
+And that wasn't even the closest map during the weekend! That map would come right after these 2 break points. On [Azumi Takahashi feat. Lotus Juice - It's Going Down Now](https://osu.ppy.sh/beatmapsets/2428514) (RC1) the Japanese players managed to all keep their combos, and after another unfortunate miss happening on the Thai side, it looked like Japan would run away with yet another break point. But as the map came to a close, and with the accuracy lead on their side, Thailand managed to take the point by the skin of their teeth.
+
+To round off the closest maps of this weekend we have scores from the matches between ::{ flag=PL }:: Poland and the ::{ flag=PH }:: Philippines as well as between ::{ flag=HK }:: Hong Kong and the ::{ flag=GB }:: United Kingdom. Both of these matches had some incredibly close calls that deserve a watch. If you haven't read the highlights up above for these matches, be sure to go back and give them a read. We promise that it will be worth your time!
+
```

Comment:
this section is already very long and just concluded a neatly packaged story of thailand and japan's close scores. this paragraph doesn't seem to add much

```suggestion
```

Reply:
Sure, no issue trimming it.